assets/i18n/strings_fa.i18n.json

```diff
@@ -0,0 +1,400 @@
+    "canceled": "فرستنده درخواست را لغو کرد"
+  },
+  "receiveOptionsPage": {
+    "title": "گزینه ها",
+    "destination": "@:settingsTab.receive.destination",
+    "appDirectory": "(پوشه لوکال سند)",
+    "saveToGallery": "@:settingsTab.receive.saveToGallery",
+    "saveToGalleryOff": "به دلیل وجود مسیر ها به طور خودکار خاموش می شود"
+  },
+  "sendPage": {
+    "waiting": "در انتظار پاسخ...",
+    "rejected": "گیرنده درخواست را رد کرد",
+    "busy": "گیرنده با درخواست دیگری مشغول است"
+  },
+  "progressPage": {
+    "titleSending": "ارسال فایل ها",
+    "titleReceiving": "دریافت فایل ها",
+    "savedToGallery": "درعکس ها ذخیره شد",
+    "total": {
+      "title": {
+        "sending": "مجموع پیشرفت ({time})",
+        "finishedError": "با خطا تمام شد",
+        "canceledSender": "توسط فرستنده لغو شد",
+        "canceledReceiver": "توسط گیرنده لغو شد"
+      },
+      "count": "فایل ها: {curr} / {n}",
+      "size": "حجم: {curr} / {n}",
+      "speed": "سرعت: {speed}/s"
+    }
+  },
+  "webSharePage": {
+    "title": "به اشتراک گذاشتن از طریق لینک",
+    "loading": "شروع سرور...",
+    "stopping": "توقف سرور...",
+    "error": "هنگام راه اندازی سرور خطایی رخ داد",
+    "hint": "به خاطر داشته باشید که اتصال رمزگذاری نشده. فقط در صورتی از این روش استفاده کنید که گیرنده لوکال سند را نصب نکرده باشد",
+    "openLink": {
+      "one": "باز کردن لینک در مرورگر:",
+      "other": "باز کردن لینک ها در مرورگر:"
+    },
+    "requests": "درخواست ها",
+    "noRequests": "هنوز درخواستی وجود ندارد",
+    "pendingRequests": "درخواست های در حال انتظار: {n}"
+  },
+  "aboutPage": {
+    "title": "درباره لوکال سند"
+  },
+  "changelogPage": {
+    "title": "تاریخچه تغییرات  برنامه"
+  },
+  "aliasGenerator(ignoreMissing)": {
+    "@info": "Different locales may have different words, it may not match 1:1",
+    "adjectives": [
+      "ستودنی",
+      "زیبا",
+      "بزرگ",
+      "درخشان",
+      "تمیز",
+      "باهوش",
+      "باحال",
+      "بامزه",
+      "کلک",
+      "مصمم",
+      "پر انرژی",
+      "کارآمد",
+      "خارق العاده",
+      "سریع",
+      "خوب",
+      "سرحال",
+      "خوب",
+      "دلپسند",
+      "باشکوه",
+      "خوش قیافه",
+      "جذاب",
+      "مهربون",
+      "دوست داشتنی",
+      "عارف",
+      "مرتب",
+      "مودب",
+      "صبور",
+      "خوشگل",
+      "قدرتمند",
+      "پولدار",
+      "مرموز",
+      "باهوش",
+      "سرسخت",
+      "خاص",
+      "هدفمند",
+      "قوی",
+      "منظم",
+      "عاقل"
+    ],
+    "fruits": [
+      "سیب",
+      "آووکادو",
+      "موز",
+      "توت سیاه",
+      "زغال اخته",
+      "کلم بروکلی",
+      "هویچ",
+      "گیلاس",
+      "نارگیل",
+      "انگور",
+      "لیمو",
+      "کاهو",
+      "انبه",
+      "خربزه",
+      "قارچ",
+      "پیاز",
+      "نارنجی",
+      "پاپایا",
+      "هلو",
+      "گلابی",
+      "آناناس",
+      "سیب زمینی",
+      "کدو تنبل",
+      "تمشک",
+      "توت فرنگی",
+      "گوجه"
+    ],
+    "combination": "{adjective} {fruit}",
```

Comment:
Just to be sure: The combination `adjective - fruit` is correct and the adjectives and fruits can be combined without any modifications? Some languages may require modifications to the adjective depending on the fruit.